src/components/profile.jsx

```diff
@@ -1,31 +1,30 @@
 import React, { Component } from 'react';
 import Gallery from './gallery'
 import firebase from '../firebaseConfig'
-import { useReducer } from 'react';
 
 class Profile extends Component {
-    state = { 
-        id: '',
-        username: '',
-        profilePicUrl: '',
-        bio: ''
+    constructor(props) {
+    super(props)
+       this.state = { 
+            id: '',
+            username: '',
+            profilePicUrl: '',
+            bio: ''
+        }
```

Comment:
i dont think you changed anything in this file. can you revert back to original state

Reply:
oh it doesn't. can you explain what the purpose of constructor(props) and super(props) is then? I thought those were needed to pass props.

Reply:
honestly i have no idea what thats for. u can still access props by doing this.props. i think u use that when you want to change the value of the props in the child class but i could be wrong